source/time_tables/tests/departure_boards_test.cpp

```diff
@@ -146,6 +200,19 @@ BOOST_AUTO_TEST_CASE(departureboard_test1) {
         BOOST_REQUIRE_EQUAL(resp.stop_schedules(1).route().name(), "B");
         BOOST_CHECK_EQUAL(resp.stop_schedules(1).date_times_size(), 1);
     }
+
+    // No terminus_schedules on terminus for toute "A"
```

Comment:
```suggestion
    // No terminus_schedules on terminus for route "A"
```
stop2 is not the _departure_ and the _terminus_ at the same time ?
I explain my point of view, when we are already on a terminus (like stop2) the response can contains itself, nope ?

Reply:
As stop2 has arrival as well as departure time, it might be the both.
In stop_schedules, we have an object stop_schedule without date_time for a terminus.
In terminus_schedule, we don't have object terminus_schedule for a terminus.